app/views/users/phone_setup/index.html.erb

```diff
@@ -33,22 +25,30 @@
 
   <%= render 'users/shared/otp_delivery_preference_selection',
              form_obj: @new_phone_form %>
-  <% unless FeatureManagement.phone_recaptcha_enabled? %>
-    <%= link_to t('two_factor_authentication.mobile_terms_of_service'),
-                MarketingSite.messaging_practices_url,
-                class: 'display-block margin-bottom-4' %>
-  <% end %>
 
   <% if TwoFactorAuthentication::PhonePolicy.new(current_user).enabled? %>
     <%= render 'users/shared/otp_make_default_number',
                form_obj: @new_phone_form %>
   <% end %>
+
   <%= render CaptchaSubmitButtonComponent.new(
         form: f,
         action: 'phone_setup',
       ).with_content(t('forms.buttons.send_one_time_code')) %>
 <% end %>
 
+<p class="margin-top-4">
```

Comment:
Per design, this should be 5 units (2.5rem). The top margin on the button might be incorrect as well (previously-existing issue).

```suggestion
<p class="margin-top-5">
```